terraform.sh

```diff
@@ -14,26 +14,40 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+DIR_STATE_IN_CONFIGMAP=/tf-state-in
```

Comment:
may be better to use
`DIR_STATE_CONFIG_MAP`
and then matching pairs:
- `DIR_STATE_IN` and `PATH_STATE_IN`
- `DIR_STATE_OUT` and `PATH_STATE_OUT`